tutorials/cassandraOutput/src/test/resources/properties-CassandraOutputTestApp.xml

```diff
@@ -0,0 +1,34 @@
+<configuration>
```

Comment:
Normally one is kept only with property names and other to run test example. I will remove all values from one is site/conf/...